recipes/database.rb

```diff
@@ -53,8 +53,11 @@
     action [:create, :start]
   end
 
-  socket = "/var/run/mysql-#{db['instance_name']}/mysqld.sock"
+  directory '/var/run/mysqld' do
```

Comment:
Agree 👍 Didn't need it for other families so far. A proper fix would support VM restart anyways.
I mistakenly opened a PR here (GH default) instead of merging the branch into my fork but feel free to use my additions (https://github.com/joe4dev/wordpress/blob/master/CHANGELOG.md#v320-2017-01-20)